src/components/TodoList.tsx

```diff
@@ -0,0 +1,49 @@
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { TodoItem } from './TodoItem';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos : Todo[],
+  removeTodo: (value: number) => void,
+  tempTodo: Todo | null,
+  loadingTodo: number[],
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos, removeTodo, tempTodo, loadingTodo,
+}) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots

Reply:
I added, but not sure that it is correct